src/opendkim.cc

```diff
@@ -812,8 +812,6 @@ NAN_METHOD(OpenDKIM::SetOption) {
   info.GetReturnValue().Set(info.This());
 }
 
-#define MAXHDRCOUNT 10000
```

Comment:
This is still used in other places, which is fine.  You should add it again.

Reply:
@godsflaw where is it being used? I only see MAXHDRCNT being used for the dump array.

Reply:
@desttinghim https://github.com/desttinghim/node-opendkim/blob/a7af8fc24473d38d8eaccdf410e9ace171783ab2/src/opendkim.cc#L817-L818